src/Apps/CollectorProfile/Routes/Saves2/Components/StackedImageLayout.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Image } from "@artsy/palette"
+import { prepareImageURLs } from "Apps/CollectorProfile/Routes/Saves2/Utils/prepareImageURLs"
+import { FC } from "react"
+import { cropped } from "Utils/resized"
+
+interface StackedImageLayoutProps {
+  imageURLs: (string | null)[]
+}
+
+interface StackImageProps {
+  url: string | null
+  index: number
+}
+
+const LARGE_IMAGE_SIZE = 190
+const SMALL_IMAGE_SIZE = 112
+const LARGE_IMAGE_OFFSET = 4
+const SMALL_IMAGE_OFFSET = 2
+
+export const StackedImageLayout: FC<StackedImageLayoutProps> = ({
+  imageURLs,
+}) => {
+  const preparedImageURLs = prepareImageURLs(imageURLs)
+  const reversedImageURLs = preparedImageURLs.reverse()
+
+  return (
+    <Box position="relative">
+      {reversedImageURLs.map((imageURL, index) => (
+        <StackImage url={imageURL} index={index} />
+      ))}
+    </Box>
+  )
+}
+
+const StackImage: FC<StackImageProps> = ({ url, index }) => {
+  const SIZE = [SMALL_IMAGE_SIZE, LARGE_IMAGE_SIZE]
+  const OFFSET_BY_INDEX = [
+    `${SMALL_IMAGE_OFFSET * index}px`,
+    `${LARGE_IMAGE_OFFSET * index}px`,
+  ]
+
+  if (url === null) {
```

Comment:
**question (non-blocking):** Are we still planning to use the [custom icon](https://www.figma.com/file/wSV2KNz97Tyj8HkH2ZYAkB/Save-%2B-Wishlist?node-id=308%3A120308&t=e0MGCuUtSoRFTish-4) for missing images? If you're planning to add later, all good, just wanted to check. (Same question for the four-up version)

Reply:
I'll clarify this moment with designer

Reply:
good catch. I completely missed this point

Reply:
Added changes here: https://github.com/artsy/force/pull/11846/commits/52a38c93de6f4477885d3d32811f27c8acdac527

<img width="745" alt="image" src="https://user-images.githubusercontent.com/3513494/216076770-dc2ff444-4ca4-4bbf-815d-f4aa29236248.png">